### ISSUE TITLE
Update dtrace-provider dependency from 0.6.0 to 0.8.8

### ISSUE THREAD

**Issue by rod4n4m1** [ldapjs/node-ldapjs#583]
Folks, unfortunately the old **dtrace-provider@0.6.0** doesn't work on MacOS Catalina with the following error thread:

```
{ Error: Cannot find module './build/Release/DTraceProviderBindings'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
```

If you export V you can see this detail when it's installing **dtrace-provider@0.6.0**:
```
../dtrace_provider.cc:89:35: error: no matching member function for call to 'NewInstance'
    v8::Local<Object> pd = klass->NewInstance();
```

And this causes a nasty error when rebuilding it.
```
5 warnings and 1 error generated.
make: *** [Release/obj.target/DTraceProviderBindings/dtrace_provider.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:193:23)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Darwin 19.0.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/rodanami/sandbox/node_modules/ldapjs/node_modules/dtrace-provider
gyp ERR! node -v v10.17.0
gyp ERR! node-gyp -v v5.0.3
gyp ERR! not ok 
```
To overcome this issue I had to use `npm shrinkwrap` to override dependencies levels written on package-lock.json and force the use of **dtrace-provider@0.8.8**.

Thanks,

**Comment by jsumners** [comment]
Please use `ldapjs@next`.

Resolved by #466.

**Comment by jsumners** [comment]
⚠️ This issue has been locked due to age. If you have encountered a recent
problem that seems to be covered by this issue, please open a new issue.

Please include a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example)
when opening a new issue.
